### PR TITLE
Introduce flexible OSB Client constructor

### PR DESCRIPTION
**Pull request by NickyMateev**
# Introduce flexible OSB Client constructor

## Motivation

Will be needed for OSB interceptor which will call broker endpoints.

## Pull Request status

* [x] Initial implementation

### CODE REVIEW

File: pkg/util/osb.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package util
+
+import osbc "github.com/pmorie/go-open-service-broker-client/v2"
+
+func NewOSBClient(skipSsl bool) osbc.CreateFunc {
```
Comment (pankrator) [inline pkg/util/osb.go]: Add godoc.

**Comment by dpanayotov** [comment]
It says that the client supports OSB v2.11,12,13 Will it be enough in our case? v2.15 is already released. Also the import path is `github.com/kubernetes-sigs/go-open-service-broker-client`as it's moved under k8s-sigs

**Comment by KirilKabakchiev** [comment]
We should double check but since this will be the SM as a platform flow, it might be ok ?  brokers on 2.15 should work fine for calls with a 2.13 client side i think?

File: pkg/util/osb.go (hunk 1 above)
Comment (NickyMateev) [reply pkg/util/osb.go]: Of course :D

**Pull request merged**